proposals/bot-orchestrators.md

```diff
@@ -0,0 +1,182 @@
+
+
+#### Assumptions &amp; hypotheses
+_What must be true for this project to matter?_
+<!--(bullet list)-->
+
+ - The storage and retrieval bots are built and able to output test results to a dashboard.
+ - The bots are capable of accepting instructions on stdin or a unix socket or some other kind of command issuance interface.
+
+#### User workflow example
+_How would a developer or user use this new capability?_
+<!--(short paragraph)-->
+
+```
+$ ./dsr-bot-orchestrator --input my-tests.json start
+```
+
+The input file `my-tests.json` might look something like this:
+
+```
+[
+	{
+		"dealType":"storeThenRetrieve",
+		"storeThenRetrieveParameters":
+			{
+				"retrievalDelayHours":"72",  // store the file, wait 72 hrs, then try to retrieve it
+			},
+		"miner":"f0xxxx",
+		"dataToStore":"random",  // this could alternatively be a file path
+		"randomDataParameters":
+			{
+				"sizeBytes":"1073741824",  // 1 GiB
+			},
+		"schedule":
+			{
+				"startDateTime":"yyyy-mm-dd_hh:mm:ss",
+				"endDateTime":"yyyy-mm-dd_hh:mm:ss",
+				"repeatIntervalDays":"7",  // means the test is re-run every 7 days
+			},
+	}
+]
+```
+
+#### Impact
+_How would this directly contribute to web3 dev stack product-market fit?_
+
+<!--
+Explain how this addresses known challenges or opportunities.
+What awesome potential impact/outcomes/results will we see if we nail this project?
+-->
+
+ - To achieve PMF, we need storage and retrieval deals to be highly reliable (success >= 99% on first attempt)
+ - Regardless of the type of user, reliability is critical
+
+#### Leverage
+_How much would nailing this project improve our knowledge and ability to execute future projects?_
+
+<!--
+Explain the opportunity or leverage point for our subsequent velocity/impact (e.g. by speeding up development, enabling more contributors, etc)
+-->
+
+ - We need an orchestrator like this to make the dealbots useful for deal success testing
+ - The combination of dealbots and this orchestrator will give us data on deal success rates
+ - Knowledge of deal success rates will help us know where to focus our debugging efforts as we improve the Filecoin network
+
+#### Confidence
+_How sure are we that this impact would be realized? Label from [this scale](https://medium.com/@nimay/inside-product-introduction-to-feature-priority-using-ice-impact-confidence-ease-and-gist-5180434e5b15)_.
+
+<!--Explain why this rating-->
+
+Confidence = 8
+
+The reasons why this impact might not be achieved are covered in the Pre-mortem section below.
+
+
+## Project definition
+#### Brief plan of attack
+
+<!--Briefly describe the milestones/steps/work needed for this project-->
+
+#### What does done look like?
+_What specific deliverables should completed to consider this project done?_
+
+ - An orchetration program to drive the dealbots exists
+ - The orchestration program follows the general usage design described in User Workflow Example above
```

Comment:
* CI unit tests demonstrate the orchestrator does enqueue deals and does follow up on them
* The orchestrator process schedules across multiple storage/retrieval bots(maybe a stretch goal?)